src/App/App.js

```diff
@@ -1,44 +1,57 @@
 import React, { Component } from 'react';
 import './App.scss';
 import Header from '../Header/Header';
-import movieData from '../Data/movieData';
-import fakeMovieDetails from '../Data/fakeMovieDetails';
 import MovieDetails from '../MovieDetails/MovieDetails';
 import tomatillo from '../assets/tomatillo.svg';
 import MovieList from '../MovieList/MovieList';
 import MovieCard from '../MovieCard/MovieCard';
+import { fetchAllMovies, fetchSingleMovie, fetchVideo } from '../Data/apiCalls';
 import logo from '../assets/logo.svg';
-import tomatillo from '../assets/tomatillo.svg';
-
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData.movies,
-      movieDetails: fakeMovieDetails.movie,
-      pageLocation: 0
-      // movieData: [],
-      // movieDetails: [],
-      // filteredMovies: [],
-      // error: ''
+      movieData: [], //all of the movie on home page
+      movieDetails: [], //singe movie
+      filteredMovies: [],
+      pageLocation: 0,
+      // currentMovie: {},
+      isLoading: true,
+      error: '',
+
     }
   }
 
+  componentDidMount() {
+    fetchAllMovies()
+      .then(movies => this.setState({ movieData: movies.movies }))
+      .catch(err => this.setState({ error: 'Something went wrong' }))
```

Comment:
I've heard a few folks chatting about creating an error components that displays 500 errors to the user in the event of a API problem. We may want to consider this for an approach to displaying errors to users.

Reply:
Sounds like a plan. I'll chat with you more about this